### PR TITLE
Add minutes for 2020-12-17

### PR DESCRIPTION
**Pull request by RealOrangeOne**

### CODE REVIEW

File: SR2021/2020-12-17.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - Decision: Livestream is at 12pm (Australians at 10pm), end time TBC by Alistair.
+- Game
+  - Discuss after the first set of matches.
+
+### Should we pin the version of webots used for the competition?
+
+- Yes, the one released at kickstart.
+- We should still aim to support all the versions anyway.
+
+### SR(A)WN
```
Comment (PeterJCLaw) [inline SR2021/2020-12-17.md:63]: :+1: to this being an always-present agenda point as a prompt :)

**Comment by PeterJCLaw** [comment]
@RealOrangeOne maybe set the base of this PR to be the `remove-template` branch? That would clarify the diff. GitHub will automatically update the base when that PR merges, so we would still merge this one to master afterwards.

File: SR2021/2020-12-17.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2021/2020-12-17.md:63]: I like this idea :+1:  I've added a heading to the agenda doc.

**Comment by RealOrangeOne** [comment]
Merged `remove-template` before reading this, woops. Then rebase deleted the commit. Have restored original commit.

**Comment by PeterJCLaw** [comment]
I wasn't actually in this meeting, so I'm not sure I can review it really.

File: SR2021/2020-12-17.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- We think we’ve mentioned everything needed, nothings changed.
+
+## Action Points
+
+### General
+
+- Ensure automated scoring is functional
+- Work out how to send logs back to teams (other than email)
+- Document which version of webots will be run for the competition / league (update between league sessions)
```
Comment (WillB97) [inline SR2021/2020-12-17.md:73]: This point partially disagrees with the point above about pinning a version. Are we pinning the version for just the next league or the entire competition?

Comment (WillB97) [reply SR2021/2020-12-17.md:73]: This point applies to the direction of srobo/competition-simulator#251

Comment (RealOrangeOne) [reply SR2021/2020-12-17.md:73]: The idea was that we defne the version being used for each event. We'll use the kickstart version for the first league, and probably update from there.

**Review by WillB97 — APPROVED** [review]

**Review by sedders123 — APPROVED** [review]

**Pull request merged**